wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java

```diff
@@ -300,7 +300,9 @@ public static boolean getBooleanProperty(String name, boolean defaultValue) {
       DriverStation.reportError(
           "Unhandled exception instantiating robot " + robotName + " " + throwable.toString(),
           elements);
-      DriverStation.reportWarning("Robots should not quit, but yours did!", false);
+      DriverStation.reportWarning("Robots should not quit, but yours did!"
+          + " Please read the above stacktrace and verify your robot logic."
+          + " Go to https://wpilib.org/stacktrace for help debuging.", false);
```

Comment:
```suggestion
          + " Go to https://wpilib.org/stacktrace for help debugging.", false);
```

Reply:
I will be reverting this change. `debugging` is spelt correctly

https://en.wikipedia.org/wiki/Debugging
https://www.merriam-webster.com/dictionary/debug